testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/OIDCClientRegistrationTest.java

```diff
@@ -468,5 +470,50 @@ private ClientRepresentation getKeycloakClient(String clientId) {
         return ApiUtil.findClientByClientId(adminClient.realms().realm(REALM_NAME), clientId).toRepresentation();
     }
 
+    @Test
+    public void testClientWithScope() throws Exception {
+        OIDCClientRepresentation clientRep = null;
+        OIDCClientRepresentation response = null;
+        String clientScope = "phone address";
+
+        clientRep = createRep();
+        clientRep.setScope(clientScope);
+        response = reg.oidc().create(clientRep);
+
+        Set<String> clientScopes = new HashSet<>(Arrays.asList(clientScope.split(" ")));
+        Set<String> registeredClientScopes = new HashSet<>(Arrays.asList(response.getScope().split(" ")));
+        assertTrue(clientScopes.equals(registeredClientScopes));
```

Comment:
Is it possible to test retrieving client with admin REST API and doublecheck that "defaultScopes" on the client are empty? 
IMO will be good to test that, so if we do some changes in the future, we can detect regressions and clarify if the behaviour is as expected (also with relation to client policies, which we will add etc.)

Reply:
@mposolda I see. I'll add the test as you suggested.

Reply:
@tnorimat Thanks for the change. For the future, this line can be replaced with the:
```
assertEquals(clientScopes, registeredClientScopes);
```
which is more readable and better regarding possible failure reporting etc. I don't want to block this PR just because of this, so just merging it. Feel free to send another PR with improving these lines in the test if you want, but however you wish  :-)